Misc/NEWS.d/next/Library/2019-12-14-14-38-40.bpo-21600.kC4Cgh.rst

```diff
@@ -0,0 +1,2 @@
+:func:`mock.patch.stopall` stops patches created with
```

Comment:
```suggestion
:func:`mock.patch.stopall` stops patches started with
```

Nit: Started with sounds more correct to me.

Reply:
Hmm, I agree the current wording is not right, not sure how to phrase it though, as `patch.dict` does not start the patches unless `start` is called. 

What do you think of:

> Fix :func:`mock.patch.stopall` to stop active patches that were created with :func:`mock.patch.dict`.

Reply:
Adding active is good. My main thing was created sounds to include decorator and context manager patches too and hence suggested started. But given that it's NEWS entry than user facing docs I think either version is okay :)

> Fix :func:mock.patch.stopall to stop active patches that were started with :func:mock.patch.dict.